logviewer/config

```diff
@@ -1,3 +1,5 @@
+skip_index=1
 lines=100
-others=1
+others=0
```

Comment:
Why change this?

Reply:
I'd like to have only `systemd`-related units displayed, and in the next (other) PR change Webmin to logging completely to journal.

Reply:
Webmin isn't going to be able to log to a journal on the many many systems without systemd though. We should be very careful not to tie ourselves to systemd, or assume it is available.

Reply:
> Webmin isn't going to be able to log to a journal on the many many systems without systemd though. We should be very careful not to tie ourselves to systemd, or assume it is available.

That's right! Though those other systems can always use System Logs NG or RS .. But if you want, we could make this option system dependent..

Reply:
Sure ... but coming back to the original question, is this change related to the rest of the PR?

Reply:
That's right! It should be part of a future change when we first enable logging to the journal. I reverted it.